e2e-tests/pages/super_search_page.py

```diff
@@ -28,8 +28,8 @@ class CrashStatsSuperSearch(CrashStatsBasePage):
     _new_line_button_locator = (By.CSS_SELECTOR, 'button.new-line')
     _highlighted_text_locator = (By.CSS_SELECTOR, 'li[class*="highlighted"]')
     _facet_text_locator = (By.CSS_SELECTOR, '#advanced-search fieldset[id="%s"] > div:nth-child(2) span:first-child')
-    _facet_field_locator = (By.CSS_SELECTOR, '#advanced-search fieldset[id="%s"] .field')
-    _operator_text_locator = (By.CSS_SELECTOR, '#advanced-search fieldset[id="%s"] > div:nth-child(4) span')
+    _facet_field_locator = (By.CSS_SELECTOR, '#advanced-search fieldset[id="%s"] .select2-search input')
+    _operator_text_locator = (By.CSS_SELECTOR, '#advanced-se    arch fieldset[id="%s"] > div:nth-child(4) span')
```

Comment:
ouch. I was trying some changes to the locators and that badness slipped in. Good eye.